cypress/integration/channel/add_users_to_channel_spec.js

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] in dicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 5]*/
+
+describe('Add user to a channel', () => {
+    it('CS15445 Join/leave messages (including combined): Usernames are links, open profile popovers', () => {
+        // # Login as 'user-1' and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+
+        // # Select 'doloremque' channel on the left hand side
+        cy.get('#sidebarItem_saepe-5').click();
```

Comment:
I'm getting test failure when using `saepe-5` and `doloremque`.  Instead of using those channels, I suggests to create 2 test channels (see `cy.apiCreateChannel`).